graylog2-web-interface/src/components/content-packs/ContentPackEntitiesList.jsx

```diff
@@ -139,14 +140,14 @@ class ContentPackEntitiesList extends React.Component {
     );
 
     const closeShowModal = () => {
-      this.setState({ showConfigModal: false });
+      this.setState({ showConfigModal: false, showConfigModalId: undefined });
     };
 
-    const openShowModal = () => {
-      this.setState({ showConfigModal: true });
+    const openShowModal = (entityId) => {
+      this.setState({ showConfigModal: true, showConfigModalId: entityId });
     };
 
-    const showModal = (
+    const showModal = entity.id === this.state.showConfigModalId ? (
       <BootstrapModalWrapper showModal={this.state.showConfigModal}
```

Comment:
Here we can just do `showModal={!!this.state.showConfigModalId}` which will be true when it has a value and be false when it is undefined